nss-dynamic-elephants/BoilerPlateRedux
# fixed nav redirects

## Description of Proposed Changes
Fixed nav issues
- 

## Steps to Test

Outline the steps to test

```sh
git fetch --all
git checkout dv-fix-nav
code .

```


1. run json server
1. npm start to view in browser
1. click on nav (settings) icon and the current page view remains (it used to switch to home view every time you clicked on this icon)
1. drop down menu for nav selections appears and the current page view stays the same
1. click on menu selection and it redirects to the selected page the nav menu closes
1. also, check all navigation menu items, each one should redirect to their appropriate pages



  

## Impacted Areas in Application

List general components of the application that this PR will affect:

* 

## Mentions @username

Tag users that need to review this code

## Definition of Done

1. The project must be fully documented. This includes the following:
    1. Complete README that documents the steps to install the code, how to install any dependencies, or system configuration needed.
    1. Every class must be documented with purpose, author, and methods.
    1. Every method must be documented with purpose and argument list - which itself must contain a short purpose for each argument.
1. The project must be able to run fully, and without errors, on each teammate's system.
1. Fulfills every requirement.
Every line of code has been peer reviewed.
1. For projects that require unit testing, core functionality must be identified and have at least one test for each.
